README.md

```diff
@@ -122,6 +122,16 @@ To deploy manually, make sure you are logged in, run the push command and pass i
 
 That will push to cloud.gov according to the instructions in the manifest and Profile.
 
+### User roles and permissions
+
+As of October 2019, we have two user roles in the system:
+
```

Comment:
Can you add an instruction to update the [Accounts Spreadsheet](https://docs.google.com/spreadsheets/d/1VM5hSsxUgqFM6t51Ejm_EjxnbxLI-pTXF_CVxZJSekQ/edit#gid=0) when any accounts are created or changed

Reply:
👍 Done!